media/js/src/simulationOne/multipleChoiceQuestion.jsx

```diff
@@ -25,9 +25,9 @@ export const MultipleChoiceQuestion = ({ question, options, answer }) => {
             {options.map((option, index) => (
                 <div key={index}>
                     <input
-                        type="radio"
+                        type='radio'
                         id={`option-${index}`}
-                        name="options"
+                        name='options'
```

Comment:
I actually use double-quotes in JSX templates because it's basically HTML, but I suppose either way works, so it's up to you.

Reply:
Yeah I might change it later. But this is more uniform for now, once the string is between curly bracket eslint complains about single quotes so figured everything might as well be single quote